### ISSUE TITLE
Enable Google SAML

### ISSUE THREAD

**Issue by alko89** [taskforcesh/issues#124]
Currently the only setup for signip is to use Auth0. We use G-suite and want to allow users to sign up only using our company domain, we can't do this using Auth0 as it allows logging in with any gmail address.

**Comment by manast** [comment]
@alko89 I assume your question is related to the On-Premises version of Taskforce.sh, isn't it?
I am not familiar with Google SAML, but in theory, you should be able to configure it in Auth0 https://auth0.com/docs/authenticate/protocols/saml/saml-sso-integrations/configure-auth0-saml-identity-provider/configure-auth0-as-idp-for-google-g-suite 
Have you tried it?

**Comment by alko89** [comment]
Yes, its for on premise version. Haven't tried this yet, will take a while since I don't have access to SAML config, would be much easier to be able to allow it for our admin to configure it himself. Now I've created a free account on Auth0 just for this so I can login.

It would be way more convenient to allow other forms of authentication in the dashboard itself.

**Comment by manast** [comment]
SAML is very complex... if we added other forms of authentication that did not depend on Auth0 it would be quite unlikely we would support SAML.
